tests/PHPStan/Type/data/throws-phpdocs.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types = 1);
+
+namespace ThrowsPhpDocs;
+
+use LogicException;
+use RuntimeException;
+
+interface Foo
+{
+	/**
+	 * @throws RuntimeException
+	 */
+	public function throwRuntimeException();
+
+	/**
+	 * @throws RuntimeException|LogicException
```

Comment:
Thanks, you are right. I saw this syntax in some library and phpstorm supports it too. But better to be strict...

Reply:
https://docs.phpdoc.org/references/phpdoc/tags/throws.html

Reply:
@Majkl578 I've read that it I don't think it disallows `RuntimeException|LogicException`, because it is a *type* that „represent an object of the class Exception or any subclass thereof.“